### PR TITLE
Feat: Create course review cards

### PR DESCRIPTION
**Pull request by krisstern**
### Tasks completed
1. Added course page
2. Added course summary in each course page

Screenshot 2023-12-25 at 3 56 10 AM

### CODE REVIEW

File: src/Pages/Course/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  // fireBaseGetOnce,
+  firebaseReadRealTime,
+  // firebaseSet,
+  // firebaseUpdate,
+} from "../../utils/firebase-db.ts";
+// import Divider from "@mui/material/Divider";
+// import {
+//   handleSignOut,
+//   handleSignIn,
+//   dumpSignedInUser,
```
Comment (cfech) [inline src/Pages/Course/index.tsx]: please remove all commented out code

File: src/utils/firebase-db.ts [hunk 2]
```diff
@@ -33,156 +29,140 @@ const app = initializeApp(firebaseConfig);
 // Initialize firebase database and get the reference of firebase database object.
 const database = getDatabase(app);
 
-// I running npm run dev, the application will try and connect to the local firebase real time database emulator
+// if running npm run dev, the application will try and connect to the local firebase real time database emulator
+// Does not seem to be working
 // https://firebase.google.com/docs/emulator-suite/install_and_configure
-if (location.hostname === "localhost" && import.meta.env.VITE_MODE === "dev") {
```
Comment (cfech) [inline src/utils/firebase-db.ts:38]: I would leave this in to be able to connect to the emulator.

You have to start  and seed the emulator before it works. If we take this out we will be doing development against a production database which is not a best practice

File: src/utils/firebase-db.ts [hunk 3]
```diff
@@ -33,156 +29,140 @@ const app = initializeApp(firebaseConfig);
...
-  const delete_string = `test/courses/`;
-  remove(ref(database, delete_string))
-    .then(() => {
-      console.log("success");
-    })
-    .catch((error) => {
-      console.log(error);
-    });
-};
+// export const createItem = (
```
Comment (cfech) [inline src/utils/firebase-db.ts:103]: Could leave these commented methods as an example

File: src/utils/firebase-db.ts (hunk 2 above)
Comment (krisstern) [reply src/utils/firebase-db.ts:38]: I tried with or without it but it was not working, I was still connected to the online realtime database it seems, but I will but this code snippet.

**Pull request merged**